src/test/java/gov/healthit/chpl/aqa/stepDefinitions/CollectionsPagesSteps.java

```diff
@@ -48,9 +34,21 @@ public void openAcbFilterOptions() {
      */
     @Given("^I am on \"([^\"]*)\" collections page: \"([^\"]*)\"$")
     public void loadCollectionsPage(final String ptitle, final String pname) {
-      driver.get(url + "#/collections/" + pname);
-      WebDriverWait wait = new WebDriverWait(driver, TIMEOUT);
-      wait.until(ExpectedConditions.visibilityOf(CollectionsPages.mainContent(driver)));
+        getDriver().get(getUrl() + "#/collections/" + pname);
+        WebDriverWait wait = new WebDriverWait(getDriver(), TIMEOUT);
+        wait.until(ExpectedConditions.visibilityOf(CollectionsPages.mainContent(getDriver())));
+    }
+
+    /**
+     * Loads collections page in test environment.
+     * @param pageName to get page name in url
+     * @param tEnv test environment in which tests will be run
+     */
+    @Given("^I am on collections page: \"([^\"]*)\" on \"([^\"]*)\"$")
+    public void loadCollectionsPageInTestEnv(final String pageName, final String tEnv) {
```

Comment:
This says load page on test environment, but it's called to load the page in three different environments, including PROD. Technically, we don't really even _have_ a test environment

Reply:
Correct, that's true! Can you please suggest some name?

Reply:
Could you please suggest from the following names:-
browseCollectionsPageInEnv
openCollectionsPageInEnv